sonar-ws/src/main/java/org/sonarqube/ws/client/project/ChangeVisibilityRequest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarqube.ws.client.project;
+
+public class ChangeVisibilityRequest {
+  private final String project;
+  private final String visibility;
+
+  public enum Visibility {
+    PUBLIC, PRIVATE
```

Comment:
the fact the enum value is not used as such (ie. in upper case) is implementation detail which do not need to be exposed in a public enum

I've made getters default protected for consistency